components/brave_wallet_ui/components/extension/allow-add-change-network-panel/index.tsx

```diff
@@ -119,12 +142,20 @@ function AllowAddNetworkPanel (props: Props) {
         />
         <NavButton
           buttonType='confirm'
-          text={getLocale('braveWalletAllowAddNetworkApproveButton')}
-          onSubmit={onApprove}
+          text={
+            panelType === 'change'
+              ? getLocale('braveWalletAllowChangeNetworkButton')
+              : getLocale('braveWalletAllowAddNetworkApproveButton')
```

Comment:
This should be just `braveWalletAllowAddNetworkButton` or `braveWalletApproveAddNetworkButton`

Reply:
Renamed to `braveWalletAllowAddNetworkButton`